base_classes/NXdisk_chopper.nxdl.xml

```diff
@@ -24,56 +24,76 @@
 <definition xmlns="http://definition.nexusformat.org/nxdl/3.1" category="base"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
-	name="NXdisk_chopper" 
+	name="NXdisk_chopper"
 	type="group" extends="NXobject">
-	
+
+	<symbols>
+    <doc>This symbol will be used below to coordinate datasets with the same shape.</doc>
+    <symbol name="n"><doc>Number of slits in the disk</doc></symbol>
+  </symbols>
+
 	<doc>
-		A device blocking the beam in a temporal periodic pattern.
-		
-		TODO: need documentation
+		A disk which blocks the beam but has one or more slits to periodically
+		let neutrons through as the disk rotates. Often used in pairs, one
+		NXdisk_chopper should be defined for each disk.
+
+		The rotation of the disk is commonly monitored by recording a timestamp for
+		each full rotation of disk, by having a sensor in the stationary disk housing
+		sensing when it is aligned with a feature (such as a magnet) on the disk.
+		We refer to this below as the "top-dead-center signal", although the format
+		does not require that the sensor be in the conventional location at the
+		top-dead-center of the disk housing.
+
+		Angles and positive rotation speeds are measured in an anticlockwise
+		direction when facing the sample.
 	</doc>
 
-	<field name="type">
-		<doc>Type of the disk-chopper: only one from the enumerated list (match text exactly)</doc>
-		<enumeration>
-			<item value="Chopper type single" />
-			<item value="contra_rotating_pair" />
-			<item value="synchro_pair" />
-		</enumeration>
-	</field>
 	<field name="rotation_speed" type="NX_FLOAT" units="NX_FREQUENCY">
-		<doc>chopper rotation speed</doc>
+		<doc>
+			Chopper rotation speed. Positive for anticlockwise rotation when
+			facing the sample, negative otherwise.
+		</doc>
 	</field>
 	<field name="slits" type="NX_INT">
 		<doc>Number of slits</doc>
 	</field>
 	<field name="slit_angle" type="NX_FLOAT" units="NX_ANGLE">
-		<doc>angular opening</doc>
+		<doc>
+			Angle of the each edge of each slit from the position of the
```

Comment:
change `the each` to `each` and I'm good overall

Reply:
To cut this short, while I can't edit @matthew-d-jones source for the PR, I'll merge and drop the "the" in master.